docs/cmd-deps-convert.md

```diff
@@ -1,8 +1,15 @@
 # dephell deps convert
 
-Convert dependencies between formats. Dephell will automagically lock them if needed and resolve all conflicts. Dependency resolution is the strongest side of the DepHell, pip and pipenv has no any dependency resolution that would be smart enough.
+Convert dependencies between formats. Dephell will automagically lock them if needed and resolve all conflicts.
```

Comment:
Why do you drop this sentence? I know, this sentence can sound too emotional, but this is the point on which all DepHell was built: I needed smart resolver for pip/Pipenv, because they couldn't lock my dependencies. Please, tell a little bit about your motivation.

Reply:
Just seemed like it wasn't adding much value to the reader.  The goal of the page should be to tell the user how to use `deps convert`, not why to pick dephell.  I think if someone has gotten so far as to read the docs for individual commands, they will already know how awesome dephell is from reading the readme.  Just feels like "preaching to the choir" here.

Then again, maybe I'm wrong.  I'm happy to put it back if you feel it's important to reiterate here.  I certainly don't mean to make dephell seem less amazing than it is.

Reply:
Make sense. However, provide context on project philosophy also important. Let's drop it here and later move into separated section. I made #150  for it.